test_runner/fixtures/neon_fixtures.py

```diff
@@ -2661,7 +2673,10 @@ def test_output_dir(request: FixtureRequest, top_output_dir: Path) -> Iterator[P
 
     yield test_dir
 
-    allure_attach_from_dir(test_dir)
+    if pytestconfig.getoption("--cleanup-test-output"):
+        shutil.rmtree(test_dir)
```

Comment:
Could we instead remove the layer and segment files, leaving logfiles and the rest intact?
This way, we will be able to remove most of the disk space and keep the data for Allure report, hence can turn the flag on by default.